### ISSUE TITLE
[Feature] Check component accessibility at build time.

### ISSUE THREAD

**Issue by sonOfTheComet-ctrl** [AmadeusITGroup/flamme#10]
## Problem
When the classes of flamme components do not have the right accessibility level,  the project compiles fine but throws at runtime. 

## Proposed solution
Add a validation step at build time which checks the interfaces accessibility and throws early if the interface/component classes are not accessible.
